deploy/cron.js

```diff
@@ -86,10 +86,10 @@ const nextRunTime = new Date(
 	currentTime.getDate() + 1,
 	10 - currentTime.getTimezoneOffset() / 60
 );
-setInterval(() => {
+setTimeout(() => {
```

Comment:
Was it intentional to change this to `setTimeout()` instead of `setInterval()`? This will make the block only run once, instead of recurring.